src/lib/i18next.js

```diff
@@ -0,0 +1,60 @@
+// This file is part of React-Invenio-Deposit
+// Copyright (C) 2021 Graz University of Technology.
+//
+// React-Invenio-Deposit is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+// I18next dependencies
+// npm install react-i18next i18next i18next-browser-languagedetector --save
+
+import i18next from 'i18next';
+
+// import translations js
+// import { TRANSLATIONS_DE } from './translations/de/translations';
+// import { TRANSLATIONS_EN } from './translations/en/translations';
+
+// import translations json
+// (Note that you need @rollup/plugin-json to import JSON files)
+// rollup.config.js
+//  - import json from "@rollup/plugin-json";
+//  - json(),
+import TRANSLATIONS_DE from './translations/de/translations.json';
+import TRANSLATIONS_EN from './translations/en/translations.json';
```

Comment:
For now, it supports both - `.js` & `.json` format of translation files - up for discussion which one could we use in order to push and pull to transifex. also there are plugins to convert these formats.

Reply:
You mean transfix can deal with both formats?

Reply:
not exactly these two formats but for JSON it should be `ICU` and for javascript it should be `RequireJS`, and i18next does support most of them, and for the one, it does not, it has plugins to do the conversion. it depends on the need of us which format do we want to use - and it would be nice to create an account for this module and test the actual pull and push.

**related links**: [transifex formats](https://docs.transifex.com/formats/introduction) &  [i18next formats](https://www.i18next.com/overview/plugins-and-utils#i-18-n-formats)